spring-boot-project/spring-boot-test/src/test/java/org/springframework/boot/test/context/FilteredClassLoaderTests.java

```diff
@@ -16,19 +16,27 @@
 
 package org.springframework.boot.test.context;
 
+import java.net.URL;
+
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
+import org.springframework.core.io.ClassPathResource;
+
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * Tests for {@link FilteredClassLoader}.
  *
  * @author Phillip Webb
+ * @author Roy Jacobs
  */
 public class FilteredClassLoaderTests {
 
+	private static ClassPathResource TEST_RESOURCE = new ClassPathResource(
+			"org/springframework/boot/test/context/FilteredClassLoaderTestResource.txt");
```

Comment:
There's a `s` missing there (`Tests`).

Reply:
Hmm, that's odd. I'd have to revisit the test to make sure that's it actually working as expected then. Will fix.

Reply:
It's missing a s in the resource as well. All is fine, it's just a naming issue :)